<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PouringWindow</class>
 <widget class="QMainWindow" name="PouringWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pouring</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/images/Logo.png</normaloff>:/images/Logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget, QDialog
{
    background-image: url(:/images/Background1.png);
}

QPushButton
{
    color:rgb(200,200,200,200);
    background-color:rgb(180,180,180,100);
    border-radius:3;
}

QPushButton:checked
{
    color:rgb(200,200,200,200);
    background-color:rgb(140,140,250,100);
    border-radius:3;
}

QLabel
{
    color:rgb(200,200,200,200);
    background-color:rgb(150,150,150, 100);
    border-radius:3;
}

#scrollPane, #paymentsScrollPane
{
    background-color:rgb(150,150,150,100);
    border-radius:3;
}

QScrollBar:horizontal, QScrollBar:vertical
{
    height:50px;
    width:50px;
    background-color:rgb(120, 120, 120, 140);
}

QScrollBar::add-page:horizontal, QScrollBar::add-page:vertical, QScrollBar::sub-page:horizontal, QScrollBar::sub-page:vertical
{
    background:none;
}


QComboBox QAbstractItemView
{
    background-color: rgb(120, 120, 120);
    selection-background-color: rgb(90, 90, 90);
}
QComboBox
{
    background-color: rgb(120, 120, 120);
    min-height:34px;
    selection-background-color: rgb(90, 90, 90);
}

QMessageBox
{
    background-color: rgb(180, 180, 180);
    background-image: url();
}
QMessageBox QLabel
{
    color:rgb(0,0,0);
    background-color:rgb(0,0,0,0);
}
QMessageBox QPushButton
{
    color:rgb(0,0,0);
    background-color:rgb(140,140,140);
    padding: 20px;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="beerNameField">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>500</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>32</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0,0,0,0);</string>
    </property>
    <property name="text">
     <string>African Amber</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="beerCompanyField">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>500</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0,0,0,0);</string>
    </property>
    <property name="text">
     <string>Mac &amp; Jacks</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="scanningField">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>250</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0,0,0,0);</string>
    </property>
    <property name="text">
     <string>Scanning...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="beerCountField">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>250</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0,0,0,0);</string>
    </property>
    <property name="text">
     <string>1st Beer</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="volumePouredField">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>130</y>
      <width>235</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0,0,0,0);</string>
    </property>
    <property name="text">
     <string>13.2oz</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="priceField">
    <property name="geometry">
     <rect>
      <x>545</x>
      <y>128</y>
      <width>235</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>32</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0,0,0,0);</string>
    </property>
    <property name="text">
     <string>$1.74</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="cameraDisplay">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>15</y>
      <width>240</width>
      <height>180</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0,0,0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="usersComboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>230</width>
      <height>38</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox QAbstractItemView
{
	background-color: rgb(120, 120, 120);
	selection-background-color: rgb(90, 90, 90);
}
QComboBox
{
	background-color: rgb(120, 120, 120);
	min-height:34px;
	selection-background-color: rgb(90, 90, 90);
}
QScrollBar:vertical
{
	width:50px;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(80, 80, 80, 255), stop:1 rgba(105, 105, 105, 255));
}
QScrollArea QWidget
{
	background-color: rgb(110, 110, 110);
}
QButton QWidget
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(91, 91, 91, 255), stop:1 rgba(155, 155, 155, 255));
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="finishPourButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>380</y>
      <width>500</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(136, 101, 101, 150);</string>
    </property>
    <property name="text">
     <string>Finish Pour</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="backgroundTile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>250</width>
      <height>190</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="backgroundTile_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>520</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="backgroundTile_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>210</y>
      <width>520</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>stats</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="backgroundTile_4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>370</y>
      <width>520</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="backgroundTile_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>250</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>user</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="backgroundTile_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>250</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="backgroundTile_7">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>120</y>
      <width>255</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>poured</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="backgroundTile_9">
    <property name="geometry">
     <rect>
      <x>535</x>
      <y>120</y>
      <width>255</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>price</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>545</x>
      <y>180</y>
      <width>235</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0,0,0,0);</string>
    </property>
    <property name="text">
     <string>@ $2.45/pint</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>backgroundTile_7</zorder>
   <zorder>backgroundTile_9</zorder>
   <zorder>backgroundTile_5</zorder>
   <zorder>backgroundTile_6</zorder>
   <zorder>backgroundTile_3</zorder>
   <zorder>backgroundTile_4</zorder>
   <zorder>backgroundTile_2</zorder>
   <zorder>backgroundTile</zorder>
   <zorder>beerNameField</zorder>
   <zorder>beerCompanyField</zorder>
   <zorder>scanningField</zorder>
   <zorder>beerCountField</zorder>
   <zorder>volumePouredField</zorder>
   <zorder>priceField</zorder>
   <zorder>cameraDisplay</zorder>
   <zorder>usersComboBox</zorder>
   <zorder>finishPourButton</zorder>
   <zorder>label_6</zorder>
  </widget>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
